awx/main/models/rbac.py

```diff
@@ -170,7 +170,7 @@ def get_absolute_url(self, request=None):
         return reverse('api:role_detail', kwargs={'pk': self.pk}, request=request)
 
     def __contains__(self, accessor):
-        if type(accessor) == User:
+        if accessor._meta.model_name == 'user':
```

Comment:
There's another case of this type checking further down in this file. In both files, actually, I think we should go ahead and remove all instances, such that the import of `User` itself is removed. That's an easy way to catch more bugs that might be caused by this.